tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Modules/TSSModule.java

```diff
@@ -377,19 +463,33 @@ private static Result<Boolean> createTaggedTSSTagShare(ThresholdKey thresholdKey
         }
     }
 
-        public static void createTaggedTSSTagShare(ThresholdKey thresholdKey, String tssTag, String deviceTssShare, String factorPub,
-                                               int deviceTssIndex, NodeDetails nodeDetails, TorusUtils torusUtils,
-                                               ThresholdKeyCallback<Boolean> callback) throws RuntimeError, Exception {
-            thresholdKey.executor.execute(() -> {
-                try {
-                    Result<Boolean> createTagResult = createTaggedTSSTagShare(thresholdKey, deviceTssShare, factorPub, deviceTssIndex, tssTag, false, nodeDetails, torusUtils);
-                    callback.onComplete(createTagResult);
-                } catch (Exception e) {
-                    Result<Boolean> error = new Result.Error<>(e);
-                    callback.onComplete(error);
-                }
-            });
-        }
+    /**
+     * This function creates a tagged tss share.
+     * @param thresholdKey The threshold key to act on.
+     * @param tssTag The threshold key to act on.
+     * @param deviceTssShare A string representing the device tss share.
+     * @param factorPub A string representing the factor public key.
+     * @param deviceTssIndex An integer representing the device index.
+     * @param nodeDetails A NodeDetailsModel object representing the node details..
+     * @param torusUtils A TorusUtils object to be used.
+     * @param callback The method which the result will be sent to
+     * @see NodeDetails
+     * @see TorusUtils
```

Comment:
Check this links correctly in the output, since these classes reside in dependencies.

Reply:
yes added a todo, will be replace once we have java docs for torus utils and node details, we can refer those libraries document links here